IPFS_ALL_HANDS_FORMAT.md

```diff
@@ -0,0 +1,66 @@
+### IPFS All Hands Format
+
+![](https://media.giphy.com/media/JFawGLFMCJNDi/giphy.gif)
+
+## The Dream
+
+For the past couple of months, we have been discussing revamping the IPFS ALL HANDS meetings. If you want more context, please see issue https://github.com/ipfs/pm/issues/636.
+
+We want the IPFS hands-on meeting act as a space where we can showcase all of our hard work and game changing projects.The IPFS meetings give viewers a taste of some of the problems we are tackling.
+
+
+## What's in it for you? 😺
+
+The main purpose of the IPFS ALL HANDS ON CALL is to highlight the most exciting work in the broader IPFS community and disseminate high-value knowledge to the many teams improving IPFS, building on top of it, or doing relevant research. You can use this platform to:
```

Comment:
Is there a reason the name was changed to "IPFS All Hands On Call"? Previously this was an "all hands" call - aka "call" was a descriptor of this being a sync "all hands" meeting. Now it's an all hands "on call" - which sounds like a help session or by-request office hours that doesn't seem accurate, or it's an "all hands on" call - which sounds like a workshop or something. Not sure if I missed the memo about this name change or whether it's actually just an accident... =]